commands/dkcheck.ml

```diff
@@ -8,9 +8,9 @@ let eprint lc fmt =
 
 let mk_entry md e =
   match e with
-  | Decl(lc,id,st,ty) ->
+  | Decl(lc,id,pb, st,ty) ->
```

Comment:
`pb` means "problem" for me, nor "public". I think that `pub` or `priv` would be better variable names.

Reply:
I will remain it scope.